### PR TITLE
[CEN-1463] Fix get test in FA HB Payment Instruments tests suite

### PR DESCRIPTION
**Pull request by antoniocapizzi95**
This PR fixes the load test related to the get function of FA HB Payment Instruments. Previously a wrong parameter was passed in input, now everything works.

### List of changes

- Fix load test related to FA HB Payment Instruments find function (get).

### Motivation and context

In the load test related to the get (or find) function of FA HB Payment Instruments an incorrect parameter was passed as input that made the test fail, now it has been fixed.

### Aggregation level
A single test has been fixed

- [x] Test case
- [ ] Test suite

### Affected environment

- [x] Development (DEV)
- [x] User Acceptance / Third Part Integration (UAT)
- [ ] Production (PROD)

### Test type

- [ ] End to end
- [x] Performance
- [ ] Smoke test

### Type of changes

- [ ] Add new test case/suite
- [x] Modify existing test case/suite

### Test Results

- [x] Success
- [ ] Failure
- [ ] Poor performance
- [ ] Good performance

### CODE REVIEW

**Comment by GiovanniMancini** [comment]
@antoniocapizzi95 please rebase with main and resolve conflicts

File: test/common/api/cdcSogeiHealthCheck.js [hunk 1]
```diff
@@ -7,4 +7,4 @@ export function sogeiHealthCheck(baseUrl, params) {
     const res = http.get(`${baseUrl}${API_PREFIX}`, myParams)
```
Comment (GiovanniMancini) [inline test/common/api/cdcSogeiHealthCheck.js:7]: This file shouldn't be affected in this pr

File: test/performance/cdcSogei.js [hunk 2]
```diff
@@ -10,57 +10,67 @@ const REGISTERED_ENVS = [UAT]
 const services = JSON.parse(open('../../services/environments.json'))
```
Comment (GiovanniMancini) [inline test/performance/cdcSogei.js:11]: this file shouldn't be affected in this pr

File: test/performance/faHbPaymentInstruments.js [hunk 3]
```diff
@@ -0,0 +1,123 @@
...
+}
+
+// In performance tests we shall use abort() to prevent the execution
+// of the default function, otherwise the VUs will be spawned
+if (!isTestEnabledOnEnv(__ENV.TARGET_ENV, REGISTERED_ENVS)) {
+    console.log('Test not enabled for target env')
+    exec.test.abort()
+}
+
+const body = {
```
Comment (GiovanniMancini) [inline test/performance/faHbPaymentInstruments.js:74]: This body is the  body of the put FA payment instrument, we shouldn't reuse it in other tests. 
Maybe we could move in a different variable - e.g. `const encrypted_pan` - the PGP message. What do you think?

**Review by GiovanniMancini — CHANGES_REQUESTED** [review]
Thank you @antoniocapizzi95 , great work! Just a couple of minor observations: please rebase the current branch with main. We should also rearrange variables to avoid potential confusion

**Review by GiovanniMancini — APPROVED** [review]

File: test/performance/faHbPaymentInstruments.js (hunk 3 above)
Comment (antoniocapizzi95) [reply test/performance/faHbPaymentInstruments.js]: it's a good idea! Let's consider if we should make the change now or in the future pr

**Pull request merged**